docs/user/security/authentication/index.asciidoc

```diff
@@ -293,6 +294,109 @@ xpack.security.authc.providers:
 
 Kibana uses SPNEGO, which wraps the Kerberos protocol for use with HTTP, extending it to web applications. At the end of the Kerberos handshake, Kibana will forward the service ticket to Elasticsearch. Elasticsearch will unpack it and it will respond with an access and refresh token which are then used for subsequent authentication.
 
+[[anonymous-authentication]]
+==== Anonymous authentication
+
+[IMPORTANT]
+============================================================================
+Anyone with access to the network {kib} is exposed to will be able to access {kib}. Make sure that you've properly restricted the capabilities of the anonymous service account so that anonymous users can't perform destructive actions or escalate their own privileges.
+============================================================================
+
+Anonymous authentication gives users access to {kib} without requiring them to provide any sort of credentials. This can be useful if you want your users to skip the login step when you embed dashboards in another application or set up a demo {kib} instance in your internal network while still keeping other security features intact.
+
+To enable anonymous authentication in {kib}, you need to decide what credentials the anonymous service account {kib} should use internally to authenticate anonymous requests. There are three ways to specify these credentials:
+
+NOTE: You can configure only one anonymous authentication provider per {kib} instance.
```

Comment:
Suggest moving the sentence "There are three ways to specify these credentials" to after the Note.